docs/running-rules.md

```diff
@@ -9,39 +9,44 @@ import MoreHelp from "/src/components/MoreHelp"
 
 ## Introduction
 
-Rules are specific patterns based on which Semgrep reports findings in code. These findings may help you to catch issues of security, performance, correctness, and other bugs. Rules are stored in open-source Semgrep Registry that enables you to scan code without need to write anything custom.
+Rules are specific patterns based on which Semgrep reports findings in code. These findings may help you to catch issues of security, performance, correctness, and other bugs. Rules are stored in an open-source [Semgrep Registry](https://github.com/returntocorp/semgrep-rules) that enables you to scan code without need to write anything custom. Use Semgrep rules in the following ways:
 
-Contribute to the registry by writing your own rules and adding them to the <a href="https://github.com/returntocorp/semgrep-rules" target="_blank">Semgrep rules repository</a>.
+- Use existing [Semgrep Registry rules](#running-semgrep-registry-rules) or add new rules to Semgrep Registry and run them.
+- Local rules:
+  - Create and run [ephemeral rules](#ephemeral-rules) in the command line.
+  - Create and run [YAML-defined rules](#creating-and-using-yaml-defined-rules-file).
+- Run your [local rules simultaneously with Semgrep Registry rules](#running-multiple-rules-simultaneously). 
```

Comment:
I will also say, the original "Run Registry Rules" section _sort of_ had a "quick reference" section in its code block. It was confusing since it mixed the comments explaining what you could do with the code itself, however, I wonder if listing the commands below the bullet points in this new section would be helpful.

Reply:
@efortuna is the list sufficient here? Not sure. :-) This new section (do you mean: `## Running Semgrep Registry rules`?) is not trying to give a reference, but rather really say how to run Semgrep registry rules...

Reply:
yeah we're good!